src/http/io.rs

```diff
@@ -132,7 +132,7 @@ impl<T: Write> Write for Buffered<T> {
     fn write(&mut self, data: &[u8]) -> io::Result<usize> {
         let n = self.write_buf.buffer(data);
         if n == 0 {
-            Err(io::Error::from(io::ErrorKind::WouldBlock))
+            Err(io::ErrorKind::WouldBlock.into())
```

Comment:
This is the exact same thing, isn't it?

Reply:
It is, that diff line is meant to unify the styles.

Reply:
https://github.com/hyperium/hyper/pull/1303/files/1e7e1c921c151fd056e015aa831aa48deabae297#diff-af418a2cc4df8537e267b70798f23225

Reply:
https://github.com/hyperium/hyper/blob/master/CONTRIBUTING.md#git-commit-guidelines